app/lib/system/deploy.rb

```diff
@@ -5,14 +5,15 @@ module Deploy
     mattr_accessor :info
 
     class Info
-      attr_reader :revision, :deployed_at
-      attr_reader :release
+      VERSION = '2.x'
+
+      attr_reader :revision, :deployed_at, :release
 
       delegate :minor_version, :major_version, to: :version
 
       def initialize(info)
         @revision = info.fetch('revision') { `git rev-parse HEAD 2> /dev/null`.strip }
```

Comment:
`revision` still would remain as in the `.deploy_info`?

Reply:
We may override both. btw are we sure whether RHOAM has `ThreeScale.config.onpremises` configuration? As we don't want to override what they set.

Reply:
Ideally, I would like to find a way to set the commit SHA in `revision`, or maybe the image tag... But that's not really important right now, we can look into this later.

As for RHOAM, I am pretty sure they are not overriding this setting, but I'll check just in case.